monkey/infection_monkey/post_breach/trap_command/linux/trap_command.py

```diff
@@ -0,0 +1,6 @@
+def get_linux_trap_commands():
+    return [
+        'trap \'echo \"Successfully used trap command\"\' INT &&',
+        'kill -2 $$ ;',  # send SIGINT signal
+        'trap - INT'  # untrap SIGINT
```

Comment:
Just a general tip: if you're using commands/code from external sources, like ART, add a link at the top to the resource. It will both credit the creator and give us an easy reference when updating

Reply:
Alright, will keep that in mind!
(This particular one wasn't taken from an external source, though.)